metadata-analysis/Metadata-Analysis.Rmd

```diff
@@ -139,17 +164,114 @@ counts_by_week %>%
   geom_point() +
   geom_text(nudge_x = .5, color = "gray15", check_overlap = TRUE) +
   scale_color_brewer(palette = "Dark2") +
-  scale_x_date(date_breaks = "1 week", date_labels = "%m-%d") +
+  scale_x_date(date_breaks = "1 week", date_labels = "%m/%d") +
   labs(
-    title = "Eva Site Hits",
+    title = "Weekly Eva Site Hits",
     x = "Week Ending",
     y = ""
   ) +
   theme_minimal() +
+  theme(legend.position = "bottom") +
+  ylim(0, NA)
+  
+```
+
+```{r fig.width=10, fig.height=5}
+
+counts_by_month %>%
+  filter(Event == "Site hits") %>%
+  ggplot(aes(x = MonthEnding, y = n, group = Event, color = Event, label = n)) +
+  geom_line() +
+  geom_point() +
+  geom_text(nudge_x = .5, color = "gray15", check_overlap = TRUE) +
+  scale_color_brewer(palette = "Dark2") +
+  scale_x_date(
+    date_breaks = "1 month",
+    date_labels = "%B") +
+  labs(
+    title = "Monthly Eva Site Hits",
+    x = "Month Ending",
+    y = ""
+  ) +
+  theme_minimal() +
   theme(legend.position = "bottom")
   
 ```
 
+## Tabs Visited
+
+```{r fig.width=7, fig.height=6}
+
+staging_last6months %>%
+  filter(str_detect(Event, "User on") & 
+           !(Event %in% c("User on tabHome", "User on tabUpload", "User on"))
+         & !str_detect(Event, "DEMO MODE")) %>%
+  distinct(SessionID, Event, .keep_all = TRUE) %>%
+  mutate(Tab = case_when(
+    str_detect(Event, "tabChangelog") ~ "Changelog",
+    str_detect(Event, "tabClientCount") ~ "Client Counts",
+    str_detect(Event, "tabDQOrg") ~ "Organization DQ",
+    str_detect(Event, "tabDQSystem") ~ "System DQ",
+    str_detect(Event, "tabLocalSettings") ~ "Local Settings",
+    str_detect(Event, "tabPDDE") ~ "PDDE DQ",
+    TRUE ~ as.character(Event)), 
+    Tab = factor(Tab, levels = c("Local Settings", "Client Counts", "PDDE DQ", "System DQ", "Organization DQ", "Changelog"))) %>%
```

Comment:
I'm seeing a large bar in this plot for "NA" and the cause is here. You are creating a new column named Tab and populating it based on Event, but there are some cases not included, such as "User on systemExitDetail" (1) and "User on systemOverview" (192). Those would show that way in your column due to line 217, TRUE ~ as.character(Event). Then when you factor the column, you only list the categories you want to see and R populates any non-matches as "NA," thus the "NA" category on the plot. If you're meaning to filter out the Events "User on systemOverview" and "User on systemExitDetail" those should be added to the filter statement above. (Maybe list the ones you do want to see in the filter so you don't have to keep adding new ones you don't want to see.)

Reply:
I'm not seeing this issue come up